src/Text.js

```diff
@@ -13,13 +13,15 @@ export const caps = props => props.caps
 
 export const bold = props => props.bold ? { fontWeight: props.theme.bold } : null
 
+const align = responsiveStyle('text-align', 'align')
```

Comment:
You *could* DRY this up by moving this into a common utility module, but I think this is fine for now